source/manual/alerts/user-monitor.html.md

```diff
@@ -6,16 +6,13 @@ layout: manual_layout
 section: Icinga alerts
 ---
 
-This alert is triggered by the [User Monitor Jenkins job][user-monitor-job]. This task [runs a script][repo] that verifies only the correct users have access to things like GitHub,
-Sentry, AWS, Fastly and Pagerduty.
+This alert is triggered by the [User Monitor Jenkins job][user-monitor-job]. This task [runs a script][repo] that verifies only the correct users have access to things like GitHub, Sentry, AWS, Fastly and PagerDuty.
 
 ## Investigating issues
 
-When the check fails, look at the console logs for the monitor task
-[in the AWS Production Jenkins][user-monitor-job].
+When the check fails, look at the console logs for the monitor task [in the AWS Production Jenkins][user-monitor-job].
```

Comment:
Perhaps a line about the most common error is when user-reviewer and github access don't match?